packages/resto/resto.0.9/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" "Ocamlpro" ]
+license: "MIT"
+homepage: "https://gitlab.com/nomadic-labs/resto"
+bug-reports: "https://gitlab.com/nomadic-labs/resto/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/resto"
+synopsis: "A minimal OCaml library for type-safe HTTP/JSON RPCs"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" { >= "4.10" }
+  "dune" { >= "1.11" }
+  "uri" {>= "1.9.0" }
+  "json-data-encoding" {= "0.9.1" & with-test }
+  "json-data-encoding-bson" {= "0.9.1" & with-test }
+  "ezjsonm" {with-test}
+  "lwt" {with-test}
+  "base-unix"{with-test}
+  "alcotest" { = "1.5.0" & with-test}
+  "alcotest-lwt" { = "1.5.0" & with-test}
```

Comment:
```suggestion
  "alcotest" { >= "1.5.0" & with-test}
  "alcotest-lwt" { >= "1.5.0" & with-test}
```
This seems a bit strict for dependencies you don’t control, any chance those constraints could be soften?

Reply:
Sure, I updated it on sources side as well, thank you!